IBM Power documentation for HCP {please table for post GA} (#5712)
open-cluster-management/rhacm-docs: clusters/hosted_control_planes/hosted_infraenv_ibmpower.adoc

[#hosted-control-planes-create-infraenv-ibmpower]

> = Creating an InfraEnv resource for hosted control planes on x86 bare metal for IBM Power compute nodes

Reviewer: ```suggestion
= Creating an InfraEnv resource for hosted control planes on 64-bit x86 bare metal for IBM Power compute nodes
```

An `InfraEnv` is a environment where hosts that are starting the live ISO can join as agents. In this case, the agents are created in the same namespace as your hosted control plane.

> See the following procedure to create an `InfraEnv` resource:

Reviewer: ```suggestion
To create an `InfraEnv` resource, complete the following steps:
```

> . Create a YAML file to contain the configuration: See the following example:

Reviewer: ```suggestion
. Create a YAML file to contain the configuration. See the following example:
```

+
[source,yaml]
----
apiVersion: agent-install.openshift.io/v1beta1
kind: InfraEnv
metadata:
  name: ${HOSTED_CLUSTER_NAME}
  namespace: ${HOSTED_CONTROL_PLANE_NAMESPACE}

> spec:
>   pullSecretRef:

Reviewer 2: ```suggestion
spec:
  cpuArchitecture: ppc64le
  pullSecretRef:
```
